network-addr-gen-mode-dhcpall.ks.in

```diff
@@ -0,0 +1,54 @@
+#test name: network-addr-gen-mode-dhcpall
+# Test ipv6.addr-gen-mode default (rhbz#1873021) for ip=dhcp (or missing ip=) boot option
+%ksappend repos/default.ks
+
+bootloader --timeout=1
+zerombr
+clearpart --all
+autopart
+
+keyboard us
+lang en
+timezone America/New_York
```

Comment:
Could you please use fragments to set up the defaults?

Reply:
Done.

Reply:
Aaah, actually I think I can't because `fragments/shared/common/common_no_payload.ks` contains
`network --bootproto=dhcp` while I don't want any network device config by kickstart.